client/websocket/websocket.go

```diff
@@ -117,7 +117,7 @@ func (s *Server) HandleConnect(ctx context.Context, w http.ResponseWriter, r *ht
 		s.connect(ctx, wsConn, ip)
 	}()
 
-	s.log.Info("HandleConnect done.")
+	s.log.Trace("HandleConnect done.")
```

Comment:
Sorry, never should have been Info.  However, with `ws.UseLogger` added to client/core/log.go, we can get a lot more information about what the frontend is doing when it spins on market load.